src/main/java/uk/ac/ebi/embl/converter/cli/Main.java

```diff
@@ -65,25 +73,39 @@ public enum FileFormat {
         gff3
     }
 
-    @Option(names = "-f", description = "The type of the file to be converted")
+    @CommandLine.Option(
+            names = "--rules",
+            paramLabel = "<key:value,key:value>",
+            description = "Specify rules in the format key:value",
+            converter = RuleConverter.class)
+    public CliRulesOption rules;
+
+    @CommandLine.Option(names = "-f", description = "The type of the file to be converted")
     public FileFormat fromFileType;
 
-    @Option(names = "-t", description = "The type of the file to convert to")
+    @CommandLine.Option(names = "-t", description = "The type of the file to convert to")
     public FileFormat toFileType;
 
-    @Parameters(paramLabel = "[input-file]", defaultValue = "", showDefaultValue = Help.Visibility.NEVER)
+    @CommandLine.Parameters(
+            paramLabel = "[input-file]",
+            defaultValue = "",
+            showDefaultValue = CommandLine.Help.Visibility.NEVER)
     public Path inputFilePath;
 
-    @Parameters(paramLabel = "[output-file]", defaultValue = "", showDefaultValue = Help.Visibility.NEVER)
+    @CommandLine.Parameters(
+            paramLabel = "[output-file]",
+            defaultValue = "",
+            showDefaultValue = CommandLine.Help.Visibility.NEVER)
     public Path outputFilePath;
 
+    @SneakyThrows
```

Comment:
Please remove SneakyThrows, it can be used in tests and not good to use in production code.  It kind of hides the exception thrown.